WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -120,7 +120,7 @@ public static void viewCurrentBlogThemes(Context context) {
     public static void viewCurrentBlogPeople(Context context) {
         Intent intent = new Intent(context, PeopleManagementActivity.class);
         slideInFromRight(context, intent);
-        // TODO: add -> AnalyticsTracker.track(AnalyticsTracker.Stat.OPENED_PEOPLE_MANAGEMENT);
+        AnalyticsUtils.trackWithCurrentBlogDetails(AnalyticsTracker.Stat.OPENED_PEOPLE);
```

Comment:
Can we use the old name for this enum `OPENED_PEOPLE_MANAGEMENT`. It feels more clear to me what's used for.